meta/data.go

```diff
@@ -464,12 +464,17 @@ func (data *Data) UpdateUser(name, hash string) error {
 }
 
 // SetPrivilege sets a privilege for a user on a database.
-func (data *Data) SetPrivilege(name, database string, p influxql.Privilege) error {
+func (data *Data) SetPrivilege(name, database string, p influxql.Privilege, admin bool) error {
 	ui := data.User(name)
 	if ui == nil {
 		return ErrUserNotFound
 	}
 
+	if database == "" {
```

Comment:
This needs a comment. It is not clear from the code why if `database == ""` then `ui.Admin` is set. The concepts seems unrelated.